### DISCUSSION TITLE
Please add a update for 25931 insider build and 25936 insider build

### DISCUSSION THREAD

**Discussion by aldeniangman** [valinet/ExplorerPatcher#1946] [Ideas]
WHY WON'T ANYONE LISTEN TO ME?
don't say a reason why I am running an insider build

**Comment by Poopooracoocoo** [comment]
>ExplorerPatcher running on Windows Insider Preview builds is not supported. The reason is because code there changes so often that developing and maintaining patches against that is not feasible long term. Some features and the program may generally work, but active or long term support is not provided for these builds.

https://github.com/valinet/ExplorerPatcher/wiki/Reporting-problems
